pydicer/input/tcia.py

```diff
@@ -0,0 +1,93 @@
+import os
+import urllib.request, urllib.error, urllib.parse
+import urllib.request, urllib.parse, urllib.error
+from pydicer.input.base import InputBase
```

Comment:
Starting to nitpick here, but imports should be ordered by system python libraries first, then third party-libraries, then local imports...

Reply:
Never knew about this convention, fixed for all files now!